app/scripts/views/base.js

```diff
@@ -253,27 +246,26 @@ define(function (require, exports, module) {
       return 'signin';
     },
 
-    showEphemeralMessages: function () {
-      var success = this.ephemeralMessages.get('success');
+    displayStatusMessages: function () {
```

Comment:
We have `displaySuccess`, `displayError` - `showEphemeralMessages` was the odd one out. It because `displayEphemeralMessages`, then `displayStatusMessages` since `Ephemeral` was a bit of a bunk name.